sinatra-contrib/spec/content_for_spec.rb

```diff
@@ -73,7 +73,7 @@ def render(engine, template)
   end
 
   # TODO: liquid radius markaby builder nokogiri
-  engines = %w[erb erubis haml slim]
+  engines = %w[erb erubi erubis haml slim]
```

Comment:
You should also update README, https://github.com/sinatra/sinatra/blob/master/sinatra-contrib/README.md#common-extensions

Reply:
I updated it.